test/github_changelog/formatters/markdown_test.clj

```diff
@@ -1,22 +1,22 @@
 (ns github-changelog.formatters.markdown-test
   (:require [clojure
-             [string :refer [join]]
+             [string :as str]
              [test :refer :all]]
             [github-changelog
              [markdown :as markdown]
-             [schema-generators :as sgen :refer [complete-pull complete-tag]]
-             [version-examples :refer :all]]
-            [github-changelog.formatters.markdown :as f-markdown]))
+             [schema-generators :as sgen]
+             [version-examples :as v]]
+            [github-changelog.formatters.markdown :as sut]))
 
 (deftest format-tag
-  (are [content tag] (= content (f-markdown/format-tag tag))
-    (markdown/h2 (markdown/emphasis "Major")) (complete-tag {:name "Major" :version v-major})
-    (markdown/h2 "Minor") (complete-tag {:name "Minor" :version v-minor})
-    (markdown/h3 (markdown/emphasis "Patch")) (complete-tag {:name "Patch" :version v-patch})
-    (markdown/h3 "Pre-release") (complete-tag {:name "Pre-release" :version v-pre-release})
-    (markdown/h4 (markdown/emphasis "Build")) (complete-tag {:name "Build" :version v-build})))
+  (are [content tag] (= content (sut/format-tag tag))
+    (markdown/h2 (markdown/emphasis "Major")) (sgen/complete-tag {:name "Major" :version v/major})
+    (markdown/h2 "Minor") (sgen/complete-tag {:name "Minor" :version v/minor})
+    (markdown/h3 (markdown/emphasis "Patch")) (sgen/complete-tag {:name "Patch" :version v/patch})
+    (markdown/h3 "Pre-release") (sgen/complete-tag {:name "Pre-release" :version v/pre-release})
+    (markdown/h4 (markdown/emphasis "Build")) (sgen/complete-tag {:name "Build" :version v/build})))
```

Comment:
I think it would help readability if we would alias `markdown` as `md` instead. Not necessary change.